examples/server/server.cpp

```diff
@@ -2105,7 +2105,7 @@ struct server_context {
         return true;
     }
 
-    bool launch_slot_with_task(server_slot & slot, const server_task & task) {
+    bool launch_slot_with_task(server_slot & slot, const server_task && task) {
```

Comment:
```suggestion
    bool launch_slot_with_task(server_slot & slot, server_task && task) {
```

Reply:
We also move the `params` and `prompt_tokens` from task to slot, so I think this is currently not possible

Reply:
What is not possible? With the `const` present, the moves of `params` and `prompt_tokens` are actually copies. Removing the `const` from the argument will make them proper moves.

Reply:
The problem is that the prompt_tokens will soon contains image tokens as unique_ptr, so it will require a proper move.

And even without image (the current case where we have only text tokens), I think a proper move still make sense here. This funtion is the last place where the task want to go (i.e. we expect task to be destroyed here)

Reply:
Ah now looking back at this, I realized that I'm thinking the reverted (I thought that you want to add the `const`) ; Sorry for the confusion 🥲